### PR TITLE
ensure default bundle watcher is used if none passed in

### PR DESCRIPTION
**Pull request by maleck13**
Fixes the deprovision issue found here but there is a second issue with provision that I haven't got to the bottom of
https://github.com/openshift/ansible-service-broker/pull/946#issuecomment-389166217

### CODE REVIEW

**Comment by maleck13** [comment]
ping @shawn-hurley

File: runtime/runtime.go [hunk 1]
```diff
@@ -136,6 +136,9 @@ func NewRuntime(config Configuration) {
 	defaultStateManager := state{mountLocation: config.StateMountLocation, nsTarget: config.StateMasterNamespace}
 	p := &provider{coe: cluster, ExtractedCredential: c, state: defaultStateManager}
 
+	if config.WatchBundle == nil {
+		config.WatchBundle = defaultWatchRunningBundle
```
Comment (shawn-hurley) [inline runtime/runtime.go]: can we add this to the test? make sure it does not disappear again?

**Comment by maleck13** [comment]
@shawn-hurley removed the deep equal as it seemed not to like function pointers. Needs more investigation. Is there anything else that should be checked manually here?

perhaps this issue https://github.com/golang/go/issues/8554

**Comment by shawn-hurley** [comment]
I think that is correct, even testify cannot handle this case from my investigation.

**Review by shawn-hurley — APPROVED** [review]

**Comment by maleck13** [comment]
closed as resolved on master

**Pull request closed without merging**